### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by MitchellShiell**
<h1>📰 Updated with new format</h1>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,187 +1,81 @@
...
 
-Master (Release): [![Build Status](https://jenkins.qa.cancercollaboratory.org/buildStatus/icon?job=Overture.bio%2Farranger%2Fmaster)](https://jenkins.qa.cancercollaboratory.org/job/Overture.bio/job/arranger/job/master/)
+Arranger consists of three elements:
 
-> **Note**: Arranger is undergoing refactoring work, for what will become version 3+. While we do not foresee many dramatic breaking changes, an _upgrade guide_ is already in the works. Current users of v2 may look at the `legacy` branch meanwhile, where we will continue fixing newly reported bugs until further notice.
->
-> You may need to add the `--legacy-peer-deps` flag when integrating our modules into your apps. Along with the rewrite, we're steadily updating the internal dependencies so this isn't necessary.
+- **Arranger Search**, a search API layered above an Elasticsearch cluster
```
Comment (caravinci) [inline README.md:15]: This could be better labelled as the "Arranger Server". We could also include links to SQON documentation, etc.

**Comment by caravinci** [comment]
While I like the layout changes (great work, btw @MitchellShiell 🙏), there's relevant documentation being removed or relocated from the `readme` file; and given the inherent complexity in running this app as a contributor/developer, we should wait until the removed bits are available elsewhere.
Putting this on hold until then.

**Comment by MitchellShiell** [comment]
Thanks for taking a look at this Justin. Understood I'll leave the branch alone and revisit (once Arranger v3 is further along). Btw we can always relocate that info to the Wiki, more than happy to set up the basic pages/navigation ([from template](https://github.com/overture-stack/.github/issues)) that way it's ready for updated content. Let me know, I'll do my best to reduce barriers to updating documentation, if you need/think of anything feel free to reach out.

**Comment by joneubank** [comment]
Base branch is out of date and our target template has changed. Closing, but please reopen if needed.

**Pull request closed without merging**